# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ghostscript/gcc-40.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- espgs-8.15rc3/src/gp_unifs.c	2005-01-19 05:08:41.000000000 +0100
+++ espgs-8.15rc3-fixed/src/gp_unifs.c	2005-09-19 12:22:29.000000000 +0200
@@ -130,6 +129,8 @@
 /* the original version of the following code, and Richard Mlynarik */
 /* (mly@adoc.xerox.com) for an improved version. */
 
+#include <sys/dir.h>
+
 typedef struct dirstack_s dirstack;
 struct dirstack_s {
     dirstack *next;
@@ -293,7 +295,7 @@
 uint
 gp_enumerate_files_next(file_enum * pfen, char *ptr, uint maxlen)
 {
-    const dir_entry *de;
+    struct dirent *de;
     char *work = pfen->work;
     int worklen = pfen->worklen;
     char *pattern = pfen->pattern;
diff -ur espgs-8.15rc3/src/time_.h espgs-8.15rc3-fixed/src/time_.h
--- espgs-8.15rc3/src/time_.h	2005-03-08 21:46:07.000000000 +0100
+++ espgs-8.15rc3-fixed/src/time_.h	2005-09-19 12:04:49.000000000 +0200
@@ -40,9 +40,6 @@
 #else
 #  include <time.h>
 #  if !defined(__DECC) && !defined(__MWERKS__)
-struct timeval {
-    long tv_sec, tv_usec;
-};
 #  endif
 struct timezone {
     int tz_minuteswest, tz_dsttime;
